backend/svirt.pm

```diff
@@ -174,6 +175,30 @@ sub load_snapshot ($self, $args) {
     return $post_load_snapshot_command;
 }
 
+sub do_extract_assets ($self, $args) {
+    my $first_hdd = $bmwqemu::vars{S390_ZKVM} ? 'a' : 'b';
+    my $name = $args->{name};
+    die "No name given" unless $args->{name};
+    $name =~ /\.([[:alnum:]]+)$/;
+    die "No HDD specified" unless $args->{hdd_num};
+    my $hdd_num = $args->{hdd_num} - 1;
+    die "No format specified" unless $args->{format};
+    my $format = $args->{format};
+    return undef if (($format ne 'raw') and ($format ne 'qcow2'));
+    my $svirt = $self->select_console('svirt');
+    my $svirt_name = $svirt->name . chr(ord($first_hdd) + $hdd_num);
+
+    my $image_storage = '/var/lib/libvirt/images';
+    my $svirt_img_name = $image_storage . '/' . $args->{svirt_name} . '.img';
+
+    my @args = ();
+    my $cmd = "";
+    my $qemu_compress_qcow = $bmwqemu::vars{QEMU_COMPRESS_QCOW2} // 1;
```

Comment:
I know it's from the original code, but using a `QEMU...` named variable here seems odd?

Reply:
it is `qemu-img`, though

Reply:
ah ok